### PR TITLE
Adds middle and left mouse button support

### PR DESCRIPTION
**Pull request by SaphuA**
Partially implements #6.

Middle mouse to open menu. Left mouse to select item.

### CODE REVIEW

File: src/CodeConnect.Touch/TouchControl.xaml.cs [hunk 1]
```diff
@@ -78,36 +78,42 @@ public TouchControl(TouchBranchCommand entryCommand, Window parentWindow)
...
                 if (vsCommand != null)
                 {
-                    path.TouchUp += (s, e) =>
-                    {
-                        e.Handled = true;
-                        VisualStudioModule.ExecuteCommand(vsCommand.VsCommandName, vsCommand.VsCommandParams);
-                        parentWindow.Hide();
-                    };
+                    Action<RoutedEventArgs> handle = (RoutedEventArgs args) =>
+                   {
```
Comment (AmadeusW) [inline src/CodeConnect.Touch/TouchControl.xaml.cs]: Lines 82-86 need an extra space

File: src/CodeConnect.Touch/TouchControl.xaml.cs [hunk 2]
```diff
@@ -78,36 +78,42 @@ public TouchControl(TouchBranchCommand entryCommand, Window parentWindow)
...
+
+                    path.MouseLeftButtonUp += (s, e) => { handle(e, e.GetPosition(null)); };
+                    path.TouchUp += (s, e) => { handle(e, e.GetTouchPoint(null).Position); };
                 }
 
                 index++;
             }
         }
 
+        static Point windowPosition = new Point(0, 0);
```
Comment (AmadeusW) [inline src/CodeConnect.Touch/TouchControl.xaml.cs]: Move up to where other fields are and add a comment, eg `Previous position of the window, used do draw control for deeper levels in the same spot.`

**Comment by AmadeusW** [comment]
Good refactoring! The only comments regard aesthetics.

**Comment by SaphuA** [comment]
Oh whoops I thought I did those remember window changes in a different branch... I'll go and fix your comments

**Comment by SaphuA** [comment]
Done.

**Comment by AmadeusW** [comment]
Awesome! :stars:

**Pull request merged**